### PR TITLE
bump troubleshoot version

### PR DESCRIPTION
**Pull request by danbudris**
*Issue #, if available:*

*Description of changes:*
Bump troubleshoot version to latest release.

Checked the Makefile changes, no additional build flags or major modifications that would make it necessary to update the build logic we currently use AFAICT.

Tested functionality (`generate support-bundle` and `generate support-bundle-config`) of EKS-A by setting `MR_TOOLS_DISABLE=true` and using the latest release executable. 

I believe we have go 1.17 in builder base as well.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by jaxesn — CHANGES_REQUESTED** [review]
Can you add an updating section to the readme?  Look at the flux project for an example.

The build will fail and youll need to update the checksums, just pull them out of the build output.

**Comment by danbudris** [comment]
> 

yeah the flux one is very helpful generally

File: projects/replicatedhq/troubleshoot/README.md [hunk 1]
```diff
@@ -1,6 +1,54 @@
...
+
+### Manually Testing Troubleshoot Version Compatibility
+1. build the desired version locally (`make support-bundle`)
+1. copy the support-bundle binary to your path
+1. set the env var `MR_TOOLS_DISABLE=false`, so that the EKS-A commands will use local binaries
+1. test the following commands against an existing EKS-A cluster:
+    - run `eksctl-anywhere generate support-bundle-config`
+    - run `eksctl-anywehre generate support-bundle`
+
+#### Get and build Troubleshoot locally
```
Comment (jaxesn) [inline projects/replicatedhq/troubleshoot/README.md:39]: Why give these instructions? Instead why don't we just us the makefile in this repo for building locally.  ref: https://github.com/aws/eks-anywhere-build-tooling/blob/main/docs/development/building-locally.md

**Pull request merged**